github/ci/prow-deploy/files/jobs/kubevirt/kubevirt.github.io/kubevirt-github-io-postsubmits.yaml

```diff
@@ -0,0 +1,42 @@
+postsubmits:
+  kubevirt/kubevirt.github.io:
+    - name: publish-img
+      always_run: false
+      run_if_changed: "Makefile"
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-kubevirtci-quay-credential: "true"
+      cluster: phx-prow
+      spec:
+        nodeSelector:
+          type: vm
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/bash"
+              - "-c"
+              - |
+                  cat $QUAY_PASSWORD | docker login --username $(<$QUAY_USER) --password-stdin quay.io && \
+                  GITHUB_USER=kubevirt-bot GITHUB_TOKEN=$(cat /etc/github/oauth) && \
+                  make build_img && \
+                  docker push kubevirt.github.io:latest
```

Comment:
Maybe better create a dedicated tag here? I see two advantaged to the dedicated tag approach:

1. you are not automatically updating the used image for everyone, which might break the workflow
2. this will move the `latest` tag from the older image to the newer one. IIUC on `quay.io` an image that has no tag will eventually be removed

If you want to rather use a dedicated tag please look at how we do this for project-infra here: https://github.com/kubevirt/project-infra/blob/master/images/publish_image.sh

Reply:
One other thing: you are logging into quay.io, shouldn't you push against quay.io then?

Reply:
I copy/pasted from another job.  I figured standard environment vars were inherited and the command would just push to quay.io?

re tag ... I don't want to be in charge of tagging here.  How about `docker push kubevirt.github.io` and I'll make sure only one container exists in local registry and tagging happens via Makefile?

Reply:
I think if you want to push against quay, you have to explicitly write it, the default would be to push against dockerhub IIRC.

Sure, the tagging via Makefile is fine!